lib/ansible/modules/remote_management/dellemc/idrac/dellemc_idrac_server_config_profile.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell EMC OpenManage Ansible Modules
+# Version 1.3
+# Copyright (C) 2019 Dell Inc.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# All rights reserved. Dell, EMC, and other trademarks are trademarks of Dell Inc. or its subsidiaries.
+# Other trademarks may be trademarks of their respective owners.
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: dellemc_idrac_server_config_profile
+short_description: Export or Import iDRAC Server Configuration Profile (SCP).
+version_added: "2.8"
+description:
+  - Export the Server Configuration Profile (SCP) from the iDRAC or Import from a network share or a local file.
+options:
+  idrac_ip:
+    description: iDRAC IP Address.
+    type: str
+    required: True
+  idrac_user:
+    description: iDRAC username.
+    type: str
+    required: True
+  idrac_pwd:
```

Comment:
```suggestion
  idrac_password:
```

Reply:
@jamescassell We have this convention and also to maintain compatibility with our own previous release, we need the attribute name this way

Reply:
@jamescassell done changing pwd to password